docs/get-started/upgrade-guide.md

```diff
@@ -0,0 +1,20 @@
+# Upgrade Guide
```

Comment:
Please check the CHANGELOG, there were details on breakages related to move of imports, change of props etc. Should be copied here.